.gitignore

```diff
@@ -55,3 +55,9 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Image files
```

Comment:
This is just for short duration. I am working to get image array from pygame directly.